home-manager/nvim/lean.lua

```diff
@@ -1,4 +1,8 @@
-require("lspconfig")["leanls"].setup({})
+vim.lsp.config.leanls = {
+	cmd = { "lean-language-server", "--stdio" },
+	root_markers = { "lean-toolchain", "lakefile.lean" },
+}
+vim.lsp.enable("leanls")
```

Comment:
Still throws the warning but not because of my code; nixpkgs hasn't yet picked up the silencing commit. Upstream issue https://github.com/Julian/lean.nvim/issues/427